sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -58,7 +59,7 @@ class MapboxMap :
     if (nativeMap == null) {
       throw MapboxMapMemoryLeakException()
     }
-    return nativeMap
+    return nativeMap!!
```

Comment:
nit: could be simplified to
`return nativeMap ?: throw MapboxMapMemoryLeakException()`